### ISSUE TITLE
Add additional images to a task

### ISSUE THREAD

**Issue by buyiyihu** [openvinotoolkit/cvat#3072]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

There is a suggestion to develop a uploading interface to add additional images in the task detail page. Sometimes, after a task is created,  I want to add some more images. It would allow users to dealing with a growing image collection.
This may look similar to #1278 , but I think it's different.

### Current Behaviour

At the moment, add images is not supported, there is no place for adding after the task is created, 

### Possible Solution

Maybe adding a PUT method for /task/{id}/data  API for add new images

**Comment by bsekachev** [comment]
Hi @buyiyihu 

The issue is a duplicate of #95. The core team does not have current plans to support it.
Look at projects, them might be useful for you case. 
To notice issue progress, please refer to above mentioned issue.

**Comment by saumya-banthia** [comment]
Hi @bsekachev

CVAT deployment:
Using docker

**ImageName, TAG, ImageID
_cvat/server, dev, b397aa47c9f4
cvat/ui, dev, ac6ff74a0116_**

Reading through issue #95, it seemed the issue was closed and the feature implemented, please correct me if I am wrong at understanding this.

Although when I am trying to add an image to an existing task using the following code written in high-level, I encounter an error highlighted in the screenshot below the code:

```
from cvat_sdk.core.proxies.tasks import ResourceType, TasksRepo, Task
from cvat_sdk import make_client, models

task_spec = {
    "name": "table",
    "labels": [
        {
            "name": "table",
            "color": "#fa3253",
            "attributes": [],
        }
    ],
}

with make_client(host="http://localhost:8080") as client:
    # Authorize using the basic auth
    client.login(('username', 'password'))
    task_list = client.tasks.list()
    task_1 = task_list[0]
    task_1.upload_data(
        resource_type=ResourceType.LOCAL,
        resources=['index.png'],
    )
    assert task_1.size == 1
```

My understanding of the error message is that the feature is not implemented, but this comment tells me otherwise: https://github.com/opencv/cvat/issues/95#issuecomment-1221526134 

![image](https://user-images.githubusercontent.com/29239098/193242808-22139a20-a57e-4872-8a79-d0d9366fc1be.png)

**Comment by dodo124** [comment]
Hi, I just started using cvat and it is an amazing tool :-)
Thanks for your great work.
And pretty soon I stumbled over the very same issue of "adding images to a task".
Let me explain my project and workflow:

Project: Damage detection, about 10,000 images of various damages. I am working alone and am new to visual computing.

Requirements: Set up everything locally, Yolov8, cvat, conda pytorch etc.

Workflow: 
Open cvat
Create the project (with 6 cases)
Create a task1 with 100 images, where all images are in folder1
Annotate the images
Export the dataset 
Copy and paste all images and annotations into the appropriate folders in my model
Fine-tune the model

Now, in the next step, I want to do exactly the same thing, but this time with 100 additional images.
The new workflow is different and not convenient.

First, I copy 100 new images to the existing 100 images in folder1.
Then I create a task2 and upload folder1 with 200 images this time.
Then I export the annotations from task1.
Then I have to import the annotations into task2.
Then I start annotating the 100 new images.
The rest as above.

And that is why it would be more convenient to just add additional images to the existing task.

Or maybe I don't understand the workflow you use to tackle this task?

Any ideas?
